app/controllers/items/new.js

```diff
@@ -34,6 +48,64 @@ export default GoodcityController.extend({
   imageKeys: Ember.computed.localStorage(),
 
   i18n: Ember.inject.service(),
+
+  insertFixedOptions: function(column) {
+    let package_details = this.get("packageDetails");
+    switch (column) {
+      case "frequency":
+        return ["50", "50/60 (Multi)", "60", "N/A", "Other"];
+      case "voltage":
+        return [
+          "120V ~ (100-200V)",
+          "240V ~ (200-300V)",
+          "> 300V",
+          "Multi",
+          "NA",
+          "Other"
+        ];
+      case "testStatus":
+        return [
+          "Maintenance (DO NOT USE)",
+          "Tested (DO NOT USE)",
+          "Untested (DO NOT USE)"
+        ];
+      case "compTestStatus":
+        return ["Active", "Failure", "Obsolete", "Reserved", "Spares"];
+      default:
+        return [...new Set(package_details.getEach(column).filter(Boolean))];
+    }
+  },
+
+  fetchPackageDetails: Ember.computed("packageDetails", function() {
+    if (this.get("showAdditionalFields")) {
+      let package_details = this.get("packageDetails");
+      if (package_details) {
+        let subFormData = {};
+        let columns = Object.keys(package_details.get("firstObject").toJSON());
+        columns.map(column => {
+          let columnData = [];
+          columnData = this.insertFixedOptions(column);
+          subFormData[column] = columnData.map((_column, index) => {
+            return {
+              id: index + 1,
+              tag: columnData[index]
+            };
+          });
+        });
+        return subFormData;
+      }
+    }
+  }),
```

Comment:
Above property defined similarly in `app/controllers/items/detail.js`
https://github.com/crossroads/stock.goodcity/pull/523/files#diff-bd29a631641c382d388f9c48012cfe23R74
Move to some other common location like GoodcityController or use mixin/service.